platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLafOptionsPanel.java

```diff
@@ -51,9 +64,49 @@ public class FlatLafOptionsPanel extends javax.swing.JPanel {
     public FlatLafOptionsPanel(FlatLafOptionsPanelController controller) {
         this.controller = controller;
         initComponents();
+		initAccentColor();
         updateEnabled();
     }
 
+	private void initAccentColor() {
+        ArrayList<String> names = new ArrayList<>();
+        ArrayList<Color> colors = new ArrayList<>();
+        names.add("default");
+        colors.add(DEFAULT);
+
+        String s = UIManager.getString("nb.accentColors.predefined");
+        if (s == null) {
+            // FlatLaf is not the current look and feel
+            Properties properties = new Properties();
+            try {
+                properties.load(getClass().getClassLoader().getResourceAsStream(
+                        "org/netbeans/swing/laf/flatlaf/FlatLaf.properties"));
+                s = properties.getProperty("nb.accentColors.predefined");
+            } catch (IOException ex) {
+                Exceptions.printStackTrace(ex);
+            }
+        }
+
+        if (s != null) {
+            for (String part : s.split(";")) {
+                int sepIndex = part.indexOf(':');
+                if (sepIndex > 1) {
```

Comment:
Technically should be >= 1, but there are no single-character color names, so it doesn't really matter...